### PR TITLE
Update: Updated info review page

### PR DESCRIPTION
**Pull request by adodge-bcg**
# Description

Updated info review page and personal information sidecard to match confluence. Fixed small bug in user confirmation regarding saving given names.

Sidecard:
updatePersonalSideCard

Addresses on info review:
inforeviewAddresses

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Tested locally, updated snapshots to reflect content changes

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: SPDBCSC-442

### CODE REVIEW

File: src/ecrc-frontend/src/components/base/sideCard/SideCard.css [hunk 1]
```diff
@@ -94,13 +94,13 @@
 
 .dashboard-spacing {
   padding-left: 50px;
-  padding-right: 50px;
+  /* padding-right: 50px; */
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/base/sideCard/SideCard.css]: nit: remove if not required

File: src/ecrc-frontend/src/components/page/informationreview/InformationReview.js [hunk 2]
```diff
@@ -462,12 +484,11 @@ export default function InformationReview({
...
-              complete, honest and accurate. I understand that a false statement
-              or omission of facts herein may lead to a denial of a cannabis
-              workers registration. I am also aware that later discovery of an
-              omission or misrepresentation may be grounds for any finding of
-              suitability to be suspended or revoked.
+              have provided on my application is complete, honest and accurate.
+              Any false statements or deliberate omissions on a consent form
+              filed with the CRRP may result in the inability of the CRRP to
+              accurately determine whether the applicant poses a risk to
+              children or vulnerable adults.
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/informationreview/InformationReview.js]: hmm I am surprised they say this... seems like it is revealing good news to malicious people with bad intentions 😄

File: src/ecrc-frontend/src/components/base/sideCard/SideCard.css (hunk 1 above)
Comment (sdevalapurkar-bcgov) [reply src/ecrc-frontend/src/components/base/sideCard/SideCard.css]: same in one other place as well

Comment (adodge-bcg) [reply src/ecrc-frontend/src/components/base/sideCard/SideCard.css]: Good catch! Was making sure it didn't break anywhere and then forgot about them.

File: src/ecrc-frontend/src/components/page/informationreview/InformationReview.js (hunk 2 above)
Comment (adodge-bcg) [reply src/ecrc-frontend/src/components/page/informationreview/InformationReview.js]: Does feel like it needs a "and will result in application denial" or something at the end.

File: src/ecrc-frontend/src/components/base/sideCard/SideCard.css (hunk 1 above)
Comment (adodge-bcg) [reply src/ecrc-frontend/src/components/base/sideCard/SideCard.css]: Fixed

File: src/ecrc-frontend/src/components/page/informationreview/InformationReview.js (hunk 2 above)
Comment (sdevalapurkar-bcgov) [reply src/ecrc-frontend/src/components/page/informationreview/InformationReview.js]: yeah I would maybe just check with them and see what they think about adding that... might be a good idea. not a blocker obviously for this task anyways

**Pull request merged**